src/System.Management.Automation/engine/ArgumentTypeConverterAttribute.cs

```diff
@@ -152,11 +152,6 @@ internal object Transform(EngineIntrinsics engineIntrinsics, object inputData, b
                         if (_convertTypes[i] == typeof(ActionPreference))
```

Comment:
@KirkMunro, your last commit had 3 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/learning-powershell/powershell-beginners-guide.md.http://powershell.com/cs/blogs/ebookv2/default.aspx should work
```powershell
retry of URL failed with error: 
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 111
111:                                 throw "retry of URL failed with error: $($_.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/learning-powershell/README.md.http://powershell.com/cs/blogs/ebookv2/default.aspx should work
```powershell
retry of URL failed with error: 
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 111
111:                                 throw "retry of URL failed with error: $($_.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/learning-powershell/README.md.http://powershell.com/cs/media/14/default.aspx should work
```powershell
retry of URL failed with error: 
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 111
111:                                 throw "retry of URL failed with error: $($_.Message)"

```